modules/optimization/src/reporters/OptimizationData.C

```diff
@@ -315,5 +322,18 @@ OptimizationDataTempl<T>::errorCheckDataSize()
                ").");
 }
 
+template <typename T>
+Real
+OptimizationDataTempl<T>::computeMisfitNorm()
+{
+  Real val = 0.0;
+  for (auto & misfit : _misfit_values)
+    val += misfit * misfit;
+
+  // There is no Tikhonov regularization. OptimizationData does not have access to the parameter
+  // vector.
```

Comment:
We don't need this comment.  We'll move all regularization outside the reporters.
```suggestion
```